### PR TITLE
fixed path on some tcl where its pointing to wrong location

### PR DESCRIPTION
**Pull request by shot0829**

### CODE REVIEW

**Review by sungeunbae — APPROVED** [review]

**Comment by sungeunbae** [comment]
@shot0829  Why is the test failing?

**Review by sungeunbae — COMMENTED** [review]
Please review the test output

**Comment by shot0829** [comment]
> Please review the test output

wierd...it passed last time, and the only thing i added is a comment o.O

**Review by jasonmotha — CHANGES_REQUESTED** [review]
Awaiting test results. (just putting in a review to make pull-reminders happy)

File: IM_calculation/scripts/calculate_ims.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def load_args():
 
     parser = argparse.ArgumentParser(parents=[parent_parser], add_help=True)
     #
-    #parser = argparse.ArgumentParser()
```
Comment (jasonmotha) [inline IM_calculation/scripts/calculate_ims.py]: Remove these lines.

**Review by jasonmotha — CHANGES_REQUESTED** [review]
In general it looks good.

Can you list out any parts of the merge conflict that ended up being hairy?

File: IM_calculation/scripts/calculate_ims.py (hunk 1 above)
Comment (shot0829) [reply IM_calculation/scripts/calculate_ims.py]: there is one more test that fails, i'll remove all those comments after the test are all passed ( keeping them for reference just for now)

Comment (shot0829) [reply IM_calculation/scripts/calculate_ims.py]: they should be all removed now.

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**